src/main/webapp/src/pages/Masonry.css

```diff
@@ -0,0 +1,330 @@
+      height: 109px;
+      line-height: 109px; }
+    .masonry-with-flex div:nth-child(6) {
+      height: 288px;
+      line-height: 288px; }
+    .masonry-with-flex div:nth-child(7) {
+      height: 399px;
+      line-height: 399px; }
+    .masonry-with-flex div:nth-child(8) {
+      height: 230px;
+      line-height: 230px; }
+    .masonry-with-flex div:nth-child(9) {
+      height: 485px;
+      line-height: 485px; }
+    .masonry-with-flex div:nth-child(10) {
+      height: 408px;
+      line-height: 408px; }
+    .masonry-with-flex div:nth-child(11) {
+      height: 413px;
+      line-height: 413px; }
+    .masonry-with-flex div:nth-child(12) {
+      height: 425px;
+      line-height: 425px; }
+    .masonry-with-flex div:nth-child(13) {
+      height: 323px;
+      line-height: 323px; }
+    .masonry-with-flex div:nth-child(14) {
+      height: 370px;
+      line-height: 370px; }
+    .masonry-with-flex div:nth-child(15) {
+      height: 214px;
+      line-height: 214px; }
+    .masonry-with-flex div:nth-child(16) {
+      height: 199px;
+      line-height: 199px; }
+    .masonry-with-flex div:nth-child(17) {
+      height: 164px;
+      line-height: 164px; }
+    .masonry-with-flex div:nth-child(18) {
+      height: 453px;
+      line-height: 453px; }
+    .masonry-with-flex div:nth-child(19) {
+      height: 442px;
+      line-height: 442px; }
+    .masonry-with-flex div:nth-child(20) {
+      height: 339px;
+      line-height: 339px; }
+    .masonry-with-flex div:nth-child(21) {
+      height: 168px;
+      line-height: 168px; }
+    .masonry-with-flex div:nth-child(22) {
+      height: 373px;
+      line-height: 373px; }
+    .masonry-with-flex div:nth-child(23) {
+      height: 131px;
+      line-height: 131px; }
+    .masonry-with-flex div:nth-child(24) {
+      height: 256px;
+      line-height: 256px; }
+    .masonry-with-flex div:nth-child(25) {
+      height: 152px;
+      line-height: 152px; }
+    .masonry-with-flex div:nth-child(26) {
+      height: 355px;
+      line-height: 355px; }
+    .masonry-with-flex div:nth-child(27) {
+      height: 267px;
+      line-height: 267px; }
+    .masonry-with-flex div:nth-child(28) {
+      height: 190px;
+      line-height: 190px; }
+    .masonry-with-flex div:nth-child(29) {
+      height: 433px;
+      line-height: 433px; }
+    .masonry-with-flex div:nth-child(30) {
+      height: 146px;
+      line-height: 146px; }
+    .masonry-with-flex div:nth-child(31) {
+      height: 190px;
+      line-height: 190px; }
+    .masonry-with-flex div:nth-child(32) {
+      height: 156px;
+      line-height: 156px; }
+    .masonry-with-flex div:nth-child(33) {
+      height: 500px;
+      line-height: 500px; }
+    .masonry-with-flex div:nth-child(34) {
+      height: 109px;
+      line-height: 109px; }
+    .masonry-with-flex div:nth-child(35) {
+      height: 293px;
+      line-height: 293px; }
+    .masonry-with-flex div:nth-child(36) {
+      height: 169px;
+      line-height: 169px; }
+  
```

Comment:
add a new line at EOF. looks like there is an empty line, but remove spaces.

Reply:
Will add it in the next pull request.